src/templates/build-page/tbtc-sdk/UseCases.tsx

```diff
@@ -0,0 +1,113 @@
+import { FC, useRef, useState } from "react"
+import { Box, Button, SimpleGrid, Stack, Flex } from "@chakra-ui/react"
+import Card from "../../../components/Card"
+import ExternalButtonLink from "../../../components/Buttons/ExternalButtonLink"
+import { ExternalLinkHref } from "../../../components/Navbar/types"
+import { Image, ImageProps } from "../../../components"
+
+export interface CardButton {
+  label: string
+  url: string
+  variant: string
+}
+
+export interface CardCategory {
+  label: string
+}
+
+export interface UseCasesCardProps {
+  image: ImageProps
+  preTitle: string
+  title: string
+  description: string
+  categories: CardCategory[]
+  timestamp: number
+  buttons: CardButton[]
+}
+
+const UseCasesCard: FC<UseCasesCardProps> = ({
+  image,
+  preTitle,
+  title,
+  description,
+  buttons,
+}) => {
+  const [isHovered, setIsHovered] = useState(false)
+  const cardRef = useRef(null)
+
+  return (
+    <Stack spacing={6}>
+      <Box
+        ref={cardRef}
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
+        rounded="2xl"
+        p="2px"
+        bgGradient={
+          isHovered
+            ? "linear-gradient(180deg, rgba(189,48,255,0) 0%, rgba(125,0,255,1) 100%)"
+            : "none"
+        }
+      >
```

Comment:
The hover functionality is little bit over-engineered here IMO. We could just use `_hover` property from chakra. See: https://v1.chakra-ui.com/docs/styled-system/features/style-props#pseudo

This should produce the exact same outcome with less complexity:

```
      <Box
        ref={cardRef}
        rounded="2xl"
        p="2px"
        _hover={{
          bgGradient:
            "linear-gradient(180deg, rgba(189,48,255,0) 0%, rgba(125,0,255,1) 100%)",
        }}
      >
```

Don't forget to also remove the `useState` above if you agree with this.

Reply:
Addressed in https://github.com/threshold-network/website/pull/105/commits/02522d97d220ee5e52ee32525e9d80228eec79d7

Reply:
That's awesome! But let's also remove `onMouseEnter` and `onMouseLeave` from the Box component and let's also remove `useState` above:

```
const [isHovered, setIsHovered] = useState(false)
```

Reply:
Good catch, addressed in https://github.com/threshold-network/website/pull/105/commits/c13f9b5132f67d1efddaf45f0f01cbcec173b7f7